src/main/frontend/model/samples/check-hardcoded-urls.json

```diff
@@ -0,0 +1,70 @@
+{
+	"logLevel": "info",
+	"hops": [
+		{
+			"type": "declare",
+			"declarations": {
+				"path": "\"/content/swisscom/de\"",
+				"phrase": "\"https://www.swisscom.ch/content/de\"",
+				"extension": "\".html\""
+			}
+		},
+		{
+			"type": "runScript",
+			"code": "xPathQuery = str:join([\"/jcr:root\", path, \"//*[jcr:contains(., '\", phrase, \"') and jcr:contains(., '\", extension, \"') ]\"]);\n\n",
+			"extension": "jexl",
+			"putLocalsBackIntoScope": true
+		},
```

Comment:
Couldn’t this be done in a `declare` block?

Reply:
I moved the values to parameters and I'm not able to use them in a `declare` block.